station/app/local_train_minio/LocalTrainMinIO.py

```diff
@@ -1,22 +1,25 @@
-from station.app.config import settings
+#from station.app.config import settings
 from station.clients.docker.client import dockerClient
 from fastapi import UploadFile
 from station.clients.minio.client import MinioClient
 from minio.error import MinioException
+import os
 
 
 class LocalTrainMinIO:
     def __init__(self):
-        # todo remove this -> singleton minio client for api
-        settings.setup()
-        self.minio_client = MinioClient()
+        if os.getenv("MINIO_URL"):
+            None
+            #self.minio_client = MinioClient()
+        else:
+            print("no MINIO url found")
         self.docker_client = dockerClient
         # TODO do over env file
 
         self.bucket_name = "localtrain"
-        self.minio_client.add_bucket(self.bucket_name)
```

Comment:
try to find a way where not all functionality is disabled :D i.e. pass minio client to methods or initialize in mthods